.integration/tests/dpc_testnet1.rs

```diff
@@ -287,7 +287,7 @@ fn test_testnet1_dpc_execute_constraints() {
             previous_block_hash: BlockHeaderHash([0u8; 32]),
             merkle_root_hash: MerkleRootHash([0u8; 32]),
             time: 0,
-            difficulty_target: 0x07FF_FFFF_FFFF_FFFF_u64,
+            difficulty_target: 0xFFFF_FFFF_FFFF_FFFF_u64,
```

Comment:
is this value something that's tied to the testnet version? if it is, perhaps it should be some module-level `const`?

Reply:
Having gone through the whole PR it seems that it's rather a genesis-specific quality; in that case it could also be some `const` (probably in `ledger`), e.g. `GENESIS_BLOCK_DIFFICULTY` so it's more clear.

Reply:
I've removed the convention requirement for now, and will revisit this before testnet2 lands